setup.py

```diff
@@ -92,7 +92,7 @@ def get_git_sha() -> str:
         "geopy",
         "gunicorn>=20.1.0; sys_platform != 'win32'",
         "hashids>=1.3.1, <2",
-        "holidays>=0.23, <0.24",
+        "holidays>=0.28, <0.29",
```

Comment:
Typically the upper bound should be exclusive of the next major version (assuming it follows semver notation).

Reply:
@arkid15r Do you have the major version of `holidays`? If no, we can remove the upper bound or use `<1.0`

Reply:
I believe relaxing the upper bound a bit would be the best case scenario here. I'll update the PR to use `holidays>=0.28, <1.0`

@john-bodley thank you  for the suggestion.